action.yml

```diff
@@ -6,8 +6,12 @@ branding:
   color: blue
 inputs:
   file:
-    description: 'Local file to pack'
+    description: 'Local file to pack (deprecated, prefer `files`)'
     required: true
+    deprecationMessage: 'Prefer "files" supporting globs instead of "file"'
```

Comment:
I also added this message following your suggestions about deprecation, please let me know if it's fine

Reply:
Oh cool, I had no idea Actions had support for deprecations built into it.

Reply:
`file` is still marked as `required` even though it's deprecated. If we want to use `files`, what should the value of `file` be?

Reply:
There is a workflow ci.yml testing that `files` should work fine without the `file` required. Does it fail for you, @aharpervc ?

Reply:
There's a validation warning that says `file` is required. I worked around it as such:

```yaml
file: #deprecated
files: |
  path/to/files
```